### ISSUE TITLE
Version 1.0.8 World wipe

### ISSUE THREAD

**Issue by whynottodojunto** [AllTheMods/All-the-mods-7-Sky#210]
I dont have any crash issues or anything because it didnt crash but when i updated the modpack to version 1.0.8 from 1.0.7 my main and only world just got completetly deleted, i immediatly went to the files but i couldnt find it anywhere, i imagine it's imposible to get my world back but i just wanna know if its happened to anyone else or help this bug get fixed

**Comment by jeremiahwinsley** [comment]
Did you click "update to a new profile" when updating? If so, you should still have a second copy of the pack in Curseforge, the old one with your world and a new one that's a fresh install.

**Comment by whynottodojunto** [comment]
No, i dont have a second profile in curseforge of allthemods 7 to the sky
